data-imports/clean_utils.py

```diff
@@ -139,7 +139,9 @@ def clean_bbl(table, boro, block, lot):
     return sql + '''
     UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
     UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {block} = preg_replace('/[^0-9]/', '', {block});
     UPDATE {table} SET {lot} = REPLACE({lot}, ' ', 0);
+    UPDATE {table} SET {lot} = preg_replace('/[^0-9]/', '', {lot});
```

Comment:
Sorry if I misunderstood, but doesn't the line above replace ' ' with zero, therefore not covered by the regex?